pool/pool.py

```diff
@@ -564,6 +564,26 @@ async def check_and_confirm_partial(self, partial: PostPartialRequest, points_re
         except Exception as e:
             error_stack = traceback.format_exc()
             self.log.error(f"Exception in confirming partial: {e} {error_stack}")
+            
+    async def validate_payout_instructions(self, payout_instructions: str) -> List[bool, Optional[bytes32]]:
```

Comment:
This should return a tuple. Tuples should be used when there is a known and small size for the list, especially if the list has more than one type:
`Tuple[bool, bytes32]`,

Reply:
Wait why are we returning True/False, this information is redundant. If the puzzle_hash is None, that's False, if it's not None, it's True. Return type should be just Optional[bytes32].